src/components/permissions/rowLevelPolicyHelpers.ts

```diff
@@ -0,0 +1,332 @@
+import databaseConnect from '../databaseConnect'
+import { UserInfo, PermissionRow, RuleTypes } from './types'
+
+const SQLBuilder = require('json-sql-builder2')
+const jsonToSql = new SQLBuilder('PostgreSQL')
+
+// in { permissionPolicyId: 1, permissionNameId:2 }
+// out "pp1pn2"
+const getPermissionNameAbbreviation = ({ permissionPolicyId, permissionNameId }: PermissionRow) =>
+  `pp${permissionPolicyId}pn${permissionNameId}`
+
+// in { permissionPolicyId: 1, permissionNameId:2, templatePermissionId: 3 }
+// out "pp1pn2tp3"
+const getTemplatePermissionAbbreviation = (permissionRow: PermissionRow) =>
+  `${getPermissionNameAbbreviation(permissionRow)}tp${permissionRow.templatePermissionId}`
+
+// Returns new version of object with each key prefixed with prefix (and value converter to string)
+// in "prefixMe", { one: 1, two: 2}
+// out { prefixMe_one: "1", prefixMe_two: "2" }
+const remapObjectKeysWithPrefix = (prefix: string, object: Object) => {
+  const remappedObject: { [index: string]: string } = {}
+
+  Object.entries(object).forEach(([key, value]) => {
+    remappedObject[`${prefix}_${key}`] = String(value)
+  })
+
+  return remappedObject
+}
```

Comment:
Not a big deal but perhaps could move this function helpers to separated file. To make this one more focused on the final goal => to generate the row level permissions. Whereas the other functions look more like regular utility functions.

Reply:
Ok sounds good but struggling to find a good name for the file, would 'helpersUtils' be appropriate ?

Reply:
`helpersUtils` sound great

Reply:
436c4a86aa4d642c025d1d430b301194b29e9370